src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="info">
+        <legend>Personal information</legend>
+        <div class="info__data">
+          <label for="surname">Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="info__data">
+          <label for="name">Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="info__data">
+          <label for="age">How old are You?
+            <input
+              type="number"
+              name="age"
+              id="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="info__data">
+          <label for="birthday">Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+              id="birthday"
+            >
+          </label>
+        </div>
+        <div class="info__data">
+          <label for="terms">I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              id="terms"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="info">
+        <legend>Registration</legend>
+        <div class="info__data">
+          <label for="email">E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div class="info__data">
+          <label for="password">Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="info">
+        <legend>An interesting fact about you!</legend>
+        <div class="info__data">
+          <label for="catLover">Do you love cats?
+            <input
+              type="radio"
+              name="catLover"
+              id="catLover"
+              value="Yes"
+            >
+            Yes
+            <input
+              type="radio"
+              name="catLover"
+              id="catLover1"
+              value="No"
+            >
+            No
+          </label>
```

Comment:
https://user-images.githubusercontent.com/70009205/233123320-bd6c1b23-3835-462d-8a33-cd86292e6ca7.mov

Reply:
you need to insert 2 labels and 2 IDs to the inputs according to the labels
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
the second option is better

Reply:
But, correct?)